ci/tests/broker_db_lifecycle

```diff
@@ -34,7 +34,7 @@ instanceId="$(curl -s -X PUT -H "X-Broker-API-Version: 2.4" -H "Content-Type: ap
 if [[ -z ${instanceId} ]] ; then fail "Could not determine InstanceID!" ; fi
 
 echo "Bind instance..."
-URI="$(curl -s -X PUT -H "X-Broker-API-Version: 2.4" -H "Content-Type: application/json" "http://cfadmin:cfadmin@${bindIp}:8888/v2/service_instances/${instanceId}/service_bindings/${bindingId}" -d "{ \"service_id\": \"${serviceId}\", \"plan_id\": \"${planId}\", \"app_guid\": \"${appId}\"}" | jq '.credentials.uri' | tr -d '"')"
+URI="$(curl -s -X PUT -H "X-Broker-API-Version: 2.4" -H "Content-Type: application/json" "http://cfadmin:cfadmin@${bindIp}:8888/v2/service_instances/${instanceId}/service_bindings/${bindingId}" -d "{ \"service_id\": \"${serviceId}\", \"plan_id\": \"${planId}\", \"app_guid\": \"${appId}\"}" | jq -r '.credentials.uri')"
```

Comment:
The variable URI should be lower case uri unless you are exporting it to another process, which I don't see happening in this.